src/cocotb_tools/pytest/plugin.py

```diff
@@ -0,0 +1,846 @@
+            simulator’s inertial write mechanism is correct. This allows cocotb to avoid a VPI callback into Python
+            to apply writes.
+        """,
+    ),
+    Option(
+        "cocotb_user_coverage",
+        action="store_true",
+        help="""
+            Enable to collect Python coverage data for user code. For some simulators, this will also report HDL
+            coverage. If coverage configuration file doesn't exist, branch coverage is collected and files in the
+            cocotb package directory are excluded. This needs the coverage Python module to be installed.
+        """,
+    ),
+    Option(
+        "cocotb_pytest_args",
+        default_in_help="passing arguments as-is from main pytest to subprocess pytest",
+        default=[],
+        help="""
+            Override command line arguments for pytest used as regression manager (subprocess).
+            By default, it will use the same arguments as main pytest (parent process).
+            Example: pytest --capture=no -v --cocotb-pytest-args='--capture=fd -vv -rA'
+        """,
+    ),
+    Option(
+        "cocotb_pytest_dir",
+        default_in_help="current working directory",
+        metavar="PATH",
+        help="Override path from where pytest was invoked.",
+    ),
+    Option(
+        "gpi_log_level",
+        choices=("trace", "debug", "info", "warning", "error", "critical"),
+        help="""
+            The default log level of all "gpi" (the low-level simulator interface) loggers, including both Python
+            and the native GPI logger. The default is unset, which means that the log level is inherited from the
+            root logger. This behaves similarly to INFO.
+        """,
+    ),
+    Option(
+        "gpi_extra",
+        nargs="*",
+        metavar="LIBRARY:FUNCTION",
+        help="""
+            List of extra libraries that are dynamically loaded at runtime. A function from each of these libraries
+            will be called as an entry point prior to elaboration, allowing these libraries to register system
+            functions and callbacks. Note that HDL objects cannot be accessed at this time. An entry point function
+            must be named following a : separator, which follows an existing simulator convention.
+            For example: --gpi-extra libnameA.so:entryA libnameB.so:entryB will first load libnameA.so with entry
+            point entryA, then load libnameB.so with entry point entryB.
+        """,
+    ),
+    Option(
+        "pygpi_python_bin",
+        help="""
+            The Python binary in the Python environment to use with cocotb. This is set to the result of
+            cocotb-config --python-bin in the Makefiles and Python Runner. You will likely only need to set this
+            manually if you are using a Python environment other than the currently activated environment, or if you
+            are using a custom flow.
+        """,
+    ),
```

Comment:
This should always be the version of python pytest is running under. Don't give users the option to screw it up.

Reply:
@ktbarrett Blindly copied from cocotb documentation. Removed in this commit 2f6a670f6e55106da7403d9f753660b56b829ba1